tests/integration/condition__browser__have_url_test.py

```diff
@@ -19,44 +19,44 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-
-import os
-
 import pytest
 
 from selene import have
 from selene.core.exceptions import TimeoutException
-
-start_page = 'file://' + os.path.abspath(os.path.dirname(__file__)) + '/../resources/start_page.html'
+from tests.integration.helpers.givenpage import GivenPage
 
 
 def test_have_url(session_browser):
-    session_browser.open(start_page)
+    GivenPage(session_browser.driver).opened_empty()
     session_browser.should(have.url(session_browser.driver.current_url))
     session_browser.should(have.no.url(session_browser.driver.current_url[:-1]))
 
 
 def test_have_url_containing(session_browser):
-    session_browser.open(start_page)
-    session_browser.should(have.url_containing('start_page.html'))
+    GivenPage(session_browser.driver).opened_empty()
+    session_browser.should(have.url_containing('empty.html'))
     session_browser.should(have.no.url_containing('start_page.xhtml'))
 
 
 def test_fails_on_timeout_during_waiting_for_exact_url(session_browser):
     browser = session_browser.with_(timeout=0.1)
 
-    browser.open(start_page)
+    GivenPage(browser.driver).opened_empty()
 
-    with pytest.raises(TimeoutException):
+    with pytest.raises(TimeoutException) as err:
```

Comment:
is an `err` a standard common way to shorten error in python? 
if yes, then ok
but if no, let's keep it as `error` - it's yet not so long to be so economic :)

Reply:
i dunno but agree that error is better :) will do.

Reply:
Agreed. Done it in 35d0033619a81771c5f5b1bc440fe0368d198c51